.github/workflows/build.yml

```diff
@@ -76,5 +76,5 @@ jobs:
       - name: Run PennyLane shared test suite
         run: |
           cd pl
-          pytest pennylane/plugins/tests --device lightning.qubit --analytic False --skip-ops
-          pytest pennylane/plugins/tests --device lightning.qubit --analytic True --skip-ops
+          pytest pennylane/devices/tests --device lightning.qubit --analytic False --skip-ops --shots=20000
```

Comment:
We can change the shots back to the default once the test suite works in non-analytic mode.